lang_java/parsing/test_parsing_java.ml

```diff
@@ -88,6 +89,10 @@ let test_dump file =
   let str = Ocaml.string_of_v v in
   pr str
 
+let test_dump_new file =
```

Comment:
Given this, do you still want to add documentation `bblfsh` and `bblfsh-parser.py` to this PR?

Reply:
I think so? I'll just have it in case we ever want to refer back to it.